### DISCUSSION TITLE
Comparable diagrams for more than one child (like twins)

### DISCUSSION THREAD

**Discussion by Minfred** [babybuddy/babybuddy#669] [Ideas]
Hi,

i just discovered your genius program today and had a steep learning curve.
Thank you for your effort and will to share such a good thing.

I have one hint or suggestion:
I have twins and i would like to compare some of the diagrams between both child's (for example the weight development, feeding).

For parents of twins, triplets, ... its nice to have this feature to see, if one twin maybe stays behind/ need more attraction, ...)
But maybe also parents of two only kids would like to compare some data (<-- this would need "relative" time-axis' (days from birth) instead of absolute timestamp-x-axis)

Thank you for your work,
kind regards,
minfred

**Comment by cdubz** [comment]
:wave: Hi, @Minfred! Thanks for using Baby Buddy. Hope you are surviving the twins OK :sweat_smile:

I think comparison exercises like this are something I would leave to individual users. I'm weary of making comparisons between children generally so it's not something I'm keen to support directly in the app.

That said this is definitely achievable with some additional tooling and Baby Buddy's API if you are keen to dig in to it. I wonder if you could use something like Grafana for this (see #607, for example) or some other tool.

Hope that helps!

**Reply by TaddyJ** [reply]
Potentially with an API or DB access you could easily pull it into excel and do your own comparisons??
I did this for lifetime feeding type comparisons for our first child :)
